CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _When adding new entries to the changelog, please include issue/PR numbers where
 
 ## 0.11.2 (UNRELEASED)
 
+- Added support for `--output` option `kart conflicts`. [#135](https://github.com/koordinates/kart/issues/135)
```

Comment:
We should probably also mention that the feature IDs have changed in GEOJSON diff output

Reply:
Thanks for adding [it](https://github.com/koordinates/kart/commit/e168985271eca136aa12490d89753d177798cccb).